docs/docs/cmd/util/accesstoken/accesstoken-get.mdx

```diff
@@ -30,6 +30,8 @@ The `util accesstoken get` command returns an access token for the specified res
 
 If the URL of your SharePoint tenant has been previously retrieved, you can use `sharepoint` as the resource to retrieve access token for SharePoint. To verify if the URL of your SharePoint tenant has been previously retrieved, use the [`spo get`](../../spo/spo-get.mdx) command. To explicitly set the URL of your SharePoint tenant use the [`spo set`](../../spo/spo-set.mdx) command.
 
+You can use `graph` as the resource to retrieve access token for Microsoft Graph.
```

Comment:
Instead of writing this, I think it would be better to add a `tip` admonition to the docs that informs our users that they can use `sharepoint` and `graph` as valid resource.

Reply:
Do you mean :::tip section? I have added it under Remarks.

Reply:
> Do you mean :::tip section? I have added it under Remarks.

Yes indeed